src/consts/models.tsx

```diff
@@ -148,6 +148,7 @@ export enum Route {
   AirdropDashboard = 'AirdropDashboard',
   AirdropThankYou = 'AirdropThankYou',
   AirdropRequirements = 'AirdropRequirements',
+  AirdropFinished = 'AirdropFinished',
```

Comment:
`AirdropFinishedWalletDetails` maybe to be more precise

Reply:
done